code-dot-org/code-dot-org
# Create new survey rollup UI

# Description

Continue from https://github.com/code-dot-org/code-dot-org/pull/30975, this PR adds new UI component that consumes data returned by experiment API. Any change to make the UI looks better will be in another PR.

**_Facilitator roll-up tab_**
![Screen Shot 2019-09-30 at 9 47 52 AM](https://user-images.githubusercontent.com/46507039/65898908-6b216c80-e367-11e9-8547-da56991f8874.png)

**_Workshop roll-up tab_**
![Screen Shot 2019-09-30 at 9 47 40 AM](https://user-images.githubusercontent.com/46507039/65898930-75436b00-e367-11e9-9a1f-71168f25e5a2.png)


## Links



- [jira](https://codedotorg.atlassian.net/browse/PLC-318)
- [spec](https://docs.google.com/spreadsheets/d/1IU0vXpznNmd8J1F4_MdgUX_VIYT0tx6F6_lSXWpbX4o/edit?pli=1#gid=0)

## Testing story
- Use Storybook for local test.
- Use adhoc machine for testing with real production data.
  - Academic year workshops
    - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/6858
    - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/7219
  - 5-day Summer workshops
     - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/5420
    - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/5988
   - CSF Deep Dive workshops
      - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/7775
      - https://adhoc-ha-adhoc-studio.cdn-code.org/pd/workshop_dashboard/daily_survey_results/7652
  - Notes
     - You may need to create an account on the adhoc. Let me know if you need admin permission.
     - Append `?enableExperiments=rollupSurveyReport` to URL to see the new UI. Add `?disableExperiments=rollupSurveyReport` to go back to the old UI.


# Reviewer Checklist:
